Lab3.ipynb

```diff
@@ -0,0 +1,401 @@
+      "text/plain": [
+       "<Figure size 1440x1080 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "d = np.append(stats.norm.rvs(loc = 12., scale = 0.4, size = 100000), [10., 10.3, 2.1, 0., 0., 15.6, 22.3, 12.7])\n",
+    "fig, ax = plt.subplots(1, 1)\n",
+    "ax.hist(d,100, density=True)\n",
+    "plt.tick_params(labelsize = 24)\n",
+    "plt.yscale('log')\n",
+    "plt.axvline(x=11.6, c='y')\n",
+    "plt.axvline(x=12.4, c='y')\n",
+    "#x = np.linspace(4.95,5.05,1000)\n",
+    "#ax.plot(x,stats.norm.pdf(x,loc = 5., scale = 0.01),linewidth = 8,alpha = 0.7)\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "A) So our goal is to identify bad data and throw it out.\n",
+    "\n",
+    "Start by exploring the data you have created with appropriate plots\n",
+    "State the statistical question in words. Be very clear, and describe why you have chosen this statistical question.\n",
+    "Restate your question in math. Be very clear.\n",
+    "Apply your statistical test and construct a truth table.\n",
+    "True T\tBad T\n",
+    "Your Test Good\t\t\n",
+    "Your Test Bad\t\n",
+    "The idea is how often you kept the known good data (True T & Your Test Good), how often you correctly discarded bad data (Bad T & Your Test Bad), and how often mistakes were made of either omission or commission (off diagonals).\n",
+    "\n",
+    "B) How does the number of omissions, where you threw out good data, depend on the statistical threshold you chose and is it a predictable quantity?\n",
+    "\n",
+    "C) Are there mistakes of commission (bad data getting in)? If yes, are they avoidable and how do they depend on your statistical threshold?"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 19,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "31892\n"
+     ]
+    }
+   ],
+   "source": [
+    "#Statistical question: what is the probability of getting a value at or above 12.4 or at or below 11.6?\n",
```

Comment:
Make this answer a bit more prominent, like maybe put it in a markdown cell rather than a comment